packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -704,13 +704,22 @@ class AnimationController extends Animation<double>
   /// provided, [duration] will be used instead, which has to be set before [repeat] is
   /// called either in the constructor or later by using the [duration] setter.
   ///
+  /// With [repeatTimes] set as valid value, animation will be executed by given times.
+  /// If value not given, infinite animation shall be simulated.
```

Comment:
```suggestion
  /// If a value is passed to [repeatCount], the animation will perform that many
  /// iterations before stopping. Otherwise, the animation repeats indefinitely.
```